backend/YouTubeV2.Application/DTO/SubscriptionDTO.cs

```diff
@@ -1,4 +1,4 @@
 ﻿namespace YouTubeV2.Application.DTO
 {
-    public record SubscriptionDTO(Guid id, Uri avatarImage, string nickname);
+    public record SubscriptionDTO(string id, Uri avatarImage, string nickname);
```

Comment:
maybe change to SubscribtionDto for consistency with other files and spelling error